ccd-definition/CaseEventToFields/InitiateApplicationGAspec.json

```diff
@@ -54,8 +67,7 @@
     "PageID": "GAWithOrWithoutNoticePage",
     "PageDisplayOrder": 6,
```

Comment:
@pliao-hmcts could you please check the PageDisplayOrder for "generalAppInformOtherParty"?